src/Doctrine/EventListener/PublishMercureUpdatesListener.php

```diff
@@ -92,7 +92,7 @@ public function __construct(ResourceClassResolverInterface $resourceClassResolve
     public function onFlush(EventArgs $eventArgs): void
     {
         if ($eventArgs instanceof OrmOnFlushEventArgs) {
-            $uow = $eventArgs->getEntityManager()->getUnitOfWork();
```

Comment:
We should test if the `getObjectManager` method exists (and fallback on `getEntityManager` if it doesn't) for backward compatibility with old versions of Doctrine.

That's weird that it's not caught by the CI.

Reply:
PR updated